docs/NETWORK-MODELS.md

```diff
@@ -76,12 +76,31 @@ There is no connection directly to the off-device network; remote access is avai
 If an EVE device is instructed to connect ECOs via L3, it should do the following:
 
 1. Set up an L3 network on the device using a Linux bridge
+   * Checks if the existing Ethernet is not a bridge, If so rename it to **kethN** and create a bridge and name it **ethN** and move the MAC address. Reverse this actions if it's removed from DevicePortConfig(DPC)
+   * Bridges creating after DevicePortConfig changes and also after (re)boot when it receives initial config from the controller.
+   * At least one Ethernet port is assigned to EVE for management traffic
+
 1. Connect all ECOs which have this `Local` network enabled to the bridge
+
 1. If a port has been allocated to the network:
    * Set up NAT
    * Connect the designated port to the NAT
    * Connect the NAT to the bridge
 
+1. Networks can be assigned to applications using different methods
+   * app-direct which is PCI passthrough; application runs device driver
```

Comment:
Note that the term app-direct doesn't exist in EVE. The EVE API has PhyIoUsageDedicated, and that refers to other dedicated/exclusive assignment like serial ports and USB devices.